app/views/handlers/individual_response.py

```diff
@@ -225,16 +235,38 @@ def __init__(
         )
 
     def handle_get(self):
-        if self._request_args.get("journey") == "change":
+        if self._return_to_hub:
             previous_location_url = url_for(
-                "individual_response.get_individual_response_change",
-                list_item_id=self._list_item_id,
+                "individual_response.get_individual_response_who",
+                return_to=self._return_to,
             )
-        else:
+        elif self._request_args.get("journey") == "change":
```

Comment:
I think we should use `journey` instead of `return_to` when the immediate page isn't the return to location. return to makes sense if the next page or previous page would be the return to page, but `journey` was recently introduced to track the journey between multiple screens.